src/containers/App.js

```diff
@@ -112,41 +127,42 @@ class App extends Component {
         {
           //<Notification notice={ ui.notice } updateNotification={actions.updateNotification}/>
         }
-        <Notifs />
+        {/* <Notifs /> */}
```

Comment:
This `Notifs` (from `redux-notifications`) wasn't easy to just update along with the other deps... we'll need an alternative, to fork and update this package, or to roll our own little notifications lib.

Reply:
What is the exact functionality we would be loosing with this?

Reply:
Is that all notifications outside overlays?
What's the problem with `redux-notifications`?

Reply:
This is for the success/info/error notifications for the UI (not the error warnings in edit forms). 

I don't remember exactly what the error was, but think it hasn't been updated to work with the latest versions of react and/or redux. Maybe be that we can fork and update it or find another comparable package.